src/test/regress/sql/multi_extension.sql

```diff
@@ -307,3 +307,40 @@ FROM
 
 -- we don't need the schema and the function anymore
 DROP SCHEMA test_deamon CASCADE;
+
+
+-- verify citus does not crash while creating a table when run against an older worker
+-- create_distributed_table piggybacks multiple commands into single one, if one worker
+-- did not have the required UDF it should fail instead of crash.
+
+-- create a test database, configure citus with single node
+CREATE DATABASE another;
+\c - - - :worker_1_port
+CREATE DATABASE another;
+\c - - - :master_port
+
+\c another
+CREATE EXTENSION citus;
+
+SELECT FROM master_add_node('localhost', :worker_1_port);
+
+\c - - - :worker_1_port
+CREATE EXTENSION citus;
+
+-- drop function assign_distributed_transaction_id
+DELETE FROM pg_depend where objid = 'assign_distributed_transaction_id'::regproc::oid;
```

Comment:
Might be slightly cleaner to rename the function.

Reply:
yes it makes  more sense. renamed.

Reply:
for some reason travis did not like `rename to` command.

Reply:
you didn't include argument types